### ISSUE TITLE
Velocity Property does not reset

### ISSUE THREAD

**Issue by zepumph** [phetsims/balloons-and-static-electricity#278]
Found from PhET-iO work with sonification.

`balloonsAndStaticElectricity.balloonsAndStaticElectricityScreen.balloonsAndStaticElectricityModel.yellowBalloon.velocityProperty`

This property will not reset once the balloon hits the sweater and stops.  @jbphet and I will see who gets to this first, and then do a maintenance release.

**Comment by jessegreenberg** [comment]
I also ran into this for other a11y stuff.

**Comment by jessegreenberg** [comment]
In BalloonModel.applyForce, if balloon center is in charged area of sweater, this is called

```js
        if ( this.velocityProperty.get().equals( Vector2.ZERO ) ) {
          this.velocityProperty.set( Vector2.ZERO );
        }
```

The conditional looks like it is supposed to be catching this, but should be ```!```ified?

**Comment by zepumph** [comment]
@jessegreenberg feel free to go for it, I only understood the above commit minimally. Let me know if you want my help.

**Comment by jessegreenberg** [comment]
Change applied in the above commit, @zepumph or @jbphet does this seem correct and then proceed with maintenance release if necessary?

**Comment by jessegreenberg** [comment]
The velocity should also get reset when the balloon is picked up.

**Comment by jessegreenberg** [comment]
> The velocity should also get reset when the balloon is picked up.

Looks like it is getting reset when released.

**Comment by zepumph** [comment]
Makes sense to me, and things look good here. Thanks!

**Comment by jessegreenberg** [comment]
Great, thanks @zepumph!

**Comment by jbphet** [comment]
This change should be rolled into the 1.3-phetio release branch and a maintenance release should be done.  Leaving assigned to myself and @zepumph, and whoever gets to it first can do it.

**Comment by zepumph** [comment]
All done, https://phet-io.colorado.edu/sims/balloons-and-static-electricity/1.3.4-phetio/wrappers/index/

I'll tell Brianna about this new version. Closing. Thanks All!
